apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1.IntegrationTests/MetadataFixtureBase.cs

```diff
@@ -94,6 +92,11 @@ protected MetadataFixtureBase()
             emulatorProcess.Start();
             emulatorProcess.BeginOutputReadLine();
             emulatorProcess.BeginErrorReadLine();
+
+            // Block until the server becomes available.
+            var client = MetadataClient.Create();
+            while (!client.IsServerAvailable())
```

Comment:
How tight is this loop? If it's really, really tight, it might be worth having a sleep (50ms?) between each iteration. We might want a timeout, too - if it's not up in 30 seconds, we're probably in bad trouble.